test/testground/network/consensus_node.go

```diff
@@ -321,3 +326,10 @@ func parsePeerID(input string) (string, net.IP, int, error) {
 
 	return address, ip, portInt, nil
 }
+
+func safeConvert(x int) (uint16, error) {
+	if x > 0 && x <= math.MaxUint16 {
```

Comment:
```suggestion
	if x >= 0 && x <= math.MaxUint16 {
```

`uint16(0)` is a valid value

Reply:
good catch! 65e4601bccfb52143f43061d2c39c9494c93a147